.github/workflows/e2e_user_features.yml

```diff
@@ -0,0 +1,53 @@
+name: End-to-End tests
+on:
+  push:
+    branches:
+      - develop
+      - release-*
+  pull_request:
+    branches:
+      - develop
+      - release-*
+
+jobs:
+  e2e_additional_editor_and_player:
+    runs-on:  ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-18.04]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '2.x'
+          architecture: 'x64'
+      - name: Setup python by installing wheel
+        if: startsWith(github.head_ref, 'update-changelog-for-release') == false
```

Comment:
Is it possible to define this check on top and use it in other places? [not sure!] (optional)

Reply:
This is not possible to move to the top unfortunately. As far as I've been able to find, GitHub Actions YAML syntax doesn't support that

Reply:
[Future work] Can we define actions that are common in multiple workflows in one place and use it in the workflow?

https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-using-action-in-the-same-repository-as-the-workflow

Reply:
@U8NWXD any thoughts on this? Should we follow this in the future?

Reply:
yes, I think we should